doc/api-examples/go/setup.md

```diff
@@ -0,0 +1,30 @@
+<!-- markdownlint-disable-file MD013 MD036 MD041 -->
+**Setup sample project**
+
+The Go snippets require at least a simple project setup. Create it via:
+
+```bash
+# create project directory
+mkdir bluechi
+cd bluechi
+
+# initialize the project
+go mod init bluechi
+```
+
+**Installing dependencies**
+
+The Go snippets require [godbus](https://github.com/godbus/dbus/). Navigate into the `bluechi` directory and add it as a dependency via:
+
+```bash
+go get github.com/godbus/dbus/v5
+```
```

Comment:
Wouldn't it be better to just install existing RPM `golang-github-godbus-dbus-devel`?